app/controllers/api/solutions_controller.rb

```diff
@@ -93,15 +93,13 @@ def latest
 
   def update
     begin
-      solution = SolutionBase.find_by_uuid!(params[:id], current_user.id)
+      solution = SolutionBase.find_by_uuid!(params[:id])
     rescue
-      # This covers both a non-existing solution and a solution
-      # belonging to someone else. We might want seperate messages
-      # but I'm choosing not to leak the existance of a solution
-      # at this stage, but unifying the errors.
```

Comment:
^^ We're leaking this in other places anyway, so I've just removed this and am making the error messages consistent with the rest of the API